spring-boot/proxy/src/main/java/eu/coatrack/proxy/security/exceptions/AuthenticationProcessFailedException.java

```diff
@@ -20,24 +20,11 @@
  * #L%
  */
 
-import eu.coatrack.api.Proxy;
-import org.springframework.beans.factory.annotation.Value;
+import org.springframework.security.core.AuthenticationException;
 
-public class SecurityUtil {
+public class AuthenticationProcessFailedException extends AuthenticationException {
```

Comment:
There should be JavaDoc for this exception, e.g.:
```
/**
 * Indicates that an unexpected error occured during the process of authenticating
 * a call to the gateway.
 */
```
It should also allow to include the unexpected error as root cause exception.

Reply:
Both suggestions done.